input/pagecontent/ppqm.md

```diff
@@ -1,11 +1,84 @@
-According to Swiss EPR regulations, patients have the right to decide who is allowed to access and modify data in 
+### Scope
+According to Swiss EPR regulations, patients shall decide who is authorized to view the documents and data in 
 their EPR, and under which circumstances (cf. emergency access). The national integration profile "Privacy Policy 
-Query" (CH:PPQ, see Amendment 2.1 of Annex 5 EPRO-FDHA) defines how to specify these decisions as access policies in 
-the XACML 2.0 format and interchange them using the SOAP transport protocol. For mobile applications, this 
-combination of standards is only restrictedly suitable, therefore a more lightweight solution like HL7 FHIR® is 
-required instead. 
+Query" (CH:PPQ, see Amendment 2.1 of Annex 5 EPRO-FDHA) defines how this decisions are transmitted as patient privacy policies in 
+XACML 2.0 format and the SOAP transport protocol. This profile offers the same functionality based on lightweight HL7 FHIR® ressources 
+and RESTFul transactions. 
+
+### Use Cases 
+
+#### EPR Onboarding
+A patient contacts a community to open an EHR in the community. The patient passes the identification 
+process of the community and consents to the EHR. The policy administrator of the community documents 
+the patient consent by adding the patient policies to the policy repository. 
+
+The policy administrator of the community adds the following patient privacy policies to the policy repository with the 
+Mobile Privacy Policy Feed (PPQ-3) or Mobile Privacy Policy Bundle Feed (PPQ-4) transaction:
+* [EPR policy set based on template 201](http://build.fhir.org/ig/ehealthsuisse/ch-epr-fhir/StructureDefinition-PpqmConsentTemplate201.html) to grant the patient access the the EHR.
+* [EPR policy set based on template 203](http://build.fhir.org/ig/ehealthsuisse/ch-epr-fhir/StructureDefinition-PpqmConsentTemplate203.html) to set the default confidentiality level for new documents.
```

Comment:
relative link instead of absolute: StructureDefinition-PpqmConsentTemplate201.html